### ISSUE TITLE
Introduce charites instead of hocon

### ISSUE THREAD

**Issue by JinIgarashi** [unvt/naru#25]
Although charites is still under development and it’s not the sufficient level to include it in naru yet, I think it might be better to use charites instead of current hocon approach.

**Comment by hfu** [comment]
I agree. We may want to think if it would be better to create a replacement from scratch or modify naru. Thank you as always.
